pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt

```diff
@@ -327,7 +327,7 @@ class ReportManager: KoinComponent {
             this.contentType = contentType
             this.deadLetterReasons= deadLetterReasons
             this.validationSchemas= validationSchemaFileNames
-            this.content = getContent(contentType, content)
+            this.content = if (contentType!=null) getContent(contentType, content) else  null
```

Comment:
You can simplify this to:
```kotlin
this.content = contentType?.let { getContent(contentType, content) }
```
this.content will be assigned null if contentType is null and getContent() is never called if contentType is null.

Reply:
Fixed.